app/src/main/java/com/onboarding/parkingsystemkotlin/mvp/view/ParkingView.kt

```diff
@@ -11,14 +13,20 @@ import com.onboarding.parkingsystemkotlin.utils.ConstantUtils
 class ParkingView(activity: Activity) : ActivityView(activity = activity), MainActivityContract.MainActivityView {
 
     override fun showConfigureParkingLotsDialogFragment() {
-        getActivity()?.let {
+        activity?.let {
             ConfigureParkingLotsDialogFragment().show((it as AppCompatActivity).supportFragmentManager, ConstantUtils.PARKING_VIEW_TAG)
         }
     }
 
     override fun showNewReservationActivity() {
-        getActivity()?.let {
+        activity?.let {
             it.startActivity(ReserveActivity.getIntent(it))
         }
     }
+
+    override fun showReservationsRemovedToast() {
+        activity?.let {
+            Toast.makeText(activity, it.getString(R.string.main_activity_reservations_removed_toast), Toast.LENGTH_SHORT).show()
```

Comment:
here you should use it instead of activity

Reply:
done